modules/update-evaluate-availability.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-evaluate-availability_{context}"]
+= Evaluating update availability
+
+The Cluster Version Operator (CVO) periodically queries the OpenShift Update Service (OSUS) for the most recent data about update possibilities.
+This data is based on the channel to which the cluster is subscribed.
```

Comment:
I think that suggestion would break the technical accuracy of the statement. The CVO still gathers data from OSUS, but IIRC the specific data that is requested is based on the subscribed channel of the cluster. I think a sentence like this would remove the formality while still being accurate:

`This data is based on the cluster's subscribed channel`